### PR TITLE
Adding support for sleep time, ignored pools, and spare agents

### PR DESCRIPTION
**Pull request by bnookala**
Please forgive my overbearing editor for the whitespace changes. :|

I found some extra features of acs-engine-autoscaler that may be helpful, so I modified the chart's `command` field to include them. @ritazh can you do a once over on this? :)

### CODE REVIEW

**Comment by seanknox** [comment]
@k8s-bot ok to test

**Review by seanknox — APPROVED** [review]

File: stable/acs-engine-autoscaler/templates/deployment.yaml [hunk 1]
```diff
@@ -41,16 +41,28 @@ spec:
...
+            - {{ .Values.acsenginecluster.sleeptime }}
+            {{ end }}
+            {{- if .Values.acsenginecluster.ignorepools }}
+            - --ignore-pools
+            - {{ .Values.acsenginecluster.ignorepools }}
+            {{ end }}
+            {{- if .Values.acsenginecluster.spareagents }}
+            - --spare-agents
+            - {{ .Values.acsenginecluster.spareagents }}
+            {{ end }}
```
Comment (ritazh) [inline stable/acs-engine-autoscaler/templates/deployment.yaml:65]: @bnookala Seeing that this PR deals alot with whitespaces, can you pls take another look at trimming left and right whitespaces. Seems you have removed few  `{{-` . Is that intended? https://github.com/kubernetes/helm/blob/master/docs/chart_template_guide/control_structures.md

**Comment by ritazh** [comment]
Awesome work @bnookala! Thanks for adding these! Just left one comment for review.

**Comment by seanknox** [comment]
@bnookala accidentally merged this before @ritazh had a chance to review—very sorry!

**Comment by seanknox** [comment]
Hit me up if you need a follow-up PR to fix anything. Will make sure you both sign off before merging.

File: stable/acs-engine-autoscaler/templates/deployment.yaml (hunk 1 above)
Comment (bnookala) [reply stable/acs-engine-autoscaler/templates/deployment.yaml:65]: ah sorry about that. Fixed now.

**Pull request merged**